filters/xforward/xforward.go

```diff
@@ -0,0 +1,75 @@
+package xforward
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/zalando/skipper/filters"
+)
+
+const (
+	Name      = "xforward"
```

Comment:
My linter is complaining about this: exported const Name should have comment (or a comment on this block) or be unexported